extensions/cql/standard/schema/examples/example20.txt

```diff
@@ -0,0 +1,3 @@
+EXAMPLE: Find building features where the building was built before 2015.
+
+built BEFORE '2015-01-01'
```

Comment:
Should be without quotes, ie. `built BEFORE 2015-01-01`.

This applies to other cql-text examples below as well.